common/test/acceptance/pages/lms/discussion.py

```diff
@@ -733,3 +733,22 @@ def get_new_post_preview_text(self):
         """
         self.wait_for_element_visibility(".wmd-preview > div", "WMD preview pane has contents", timeout=10)
         return self.q(css=".wmd-preview").text[0]
+
+    def fill_title_field(self, text):
+        """
+        Fill title field with given text
+        """
+        self.q(css='.js-post-title').fill(text)
+
+    def get_preview_value(self):
+        """
+        Get the rendered preview of the contents of the Discussions new post editor
+        """
+        return self.q(css=".wmd-preview").html[0]
+
+    def preview_mathjax_visiblity(self):
```

Comment:
@noraiz-anwar both methods didn't satisfy my needs. but I will check the element visibility before retrieving the element.